djstripe/models.py

```diff
@@ -541,56 +406,12 @@ def charge(self, amount, currency="usd", description=None, send_receipt=True, **
         dollar amount. It will be converted to cents so any decimals beyond
         two will be ignored.
         """
-        if not isinstance(amount, decimal.Decimal):
-            raise ValueError(
-                "You must supply a decimal value representing dollars."
-            )
-        resp = Charge.api_create(
-            amount=int(amount * 100),  # Convert dollars into cents
-            currency=currency,
-            customer=self.stripe_id,
-            description=description,
-            **kwargs
-        )
-        obj = self.record_charge(resp["id"])
+        charge_id = super(Customer, self).charge(amount, currency, description, send_receipt, **kwargs)
+        obj = self.record_charge(charge_id)
```

Comment:
``obj`` is a Charge instance in this case, right? Please change the variable name to ``recorded_charge`` or whatever you think fits best